dev_guide/builds.adoc

```diff
@@ -93,6 +93,10 @@ spec:
     to:
       kind: "ImageStreamTag"
       name: "origin-ruby-sample:latest"
+  postCommit: <6>
+      command: "sh -c",
+      args: "rake test"
```

Comment:
Note that command and args are string arrays.

This example is invalid according to our API validation, as one cannot have command and script together.
I'd keep this simple and just use "script". We can describe the other possibilities further down.